archibishop/github-connect
# #156008359 Add share functionality

#### What does this PR do?
Add share functionality to the project.
#### Description of Task to be completed?
Add share functionality but adding share intent thus enabling sharing of the username and link to their GitHub page
#### How should this be manually tested?
- Clone the repository.
- Click on the on an item in the recycler view and the detail activity should be launched with the information of the item that was clicked
- Click on the share icon and choose app you want to share to.
#### What are the relevant pivotal tracker stories?
[#156008359](https://www.pivotaltracker.com/story/show/156008359)
#### Screenshots (if appropriate)
![sharefunctionailty](https://user-images.githubusercontent.com/39958516/46269683-6c00a200-c54b-11e8-8acd-db9b3bcee88e.gif)
